File: CRM/src/app/shared/eHospital_DongNai_A/BienBanHoiChan_Ekip.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { BienBanHoiChan_Ekip } from './BienBanHoiChan_Ekip.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class BienBanHoiChan_EkipService extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'BienBanHoiChan_BacSi_Id', 'BienBanHoiChan_Id', 'NhanVien_Id', 'VaiTro_Id'];

    List: BienBanHoiChan_Ekip[] | undefined;
    ListFilter: BienBanHoiChan_Ekip[] | undefined;
    FormData!: BienBanHoiChan_Ekip;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "BienBanHoiChan_Ekip";
    }
}

File: Service_eHospital_DongNai_A/Implement/BienBanHoiChan_EkipService.cs
namespace Service_eHospital_DongNai_A.Implement
{
    public class BienBanHoiChan_EkipService : BaseService<BienBanHoiChan_Ekip, IBienBanHoiChan_EkipRepository>
    , IBienBanHoiChan_EkipService
    {
    private readonly IBienBanHoiChan_EkipRepository _BienBanHoiChan_EkipRepository;
    public BienBanHoiChan_EkipService(IBienBanHoiChan_EkipRepository BienBanHoiChan_EkipRepository) : base(BienBanHoiChan_EkipRepository)
    {
    _BienBanHoiChan_EkipRepository = BienBanHoiChan_EkipRepository;
    }
    }
    }

